From 23ec35c7ab61407b5cda62a7a521385d320d05bf Mon Sep 17 00:00:00 2001
From: Felipe Borges <felipeborges@gnome.org>
Date: Thu, 10 Mar 2022 15:20:37 +0100
Subject: [PATCH 13/13] sharing: Remove leftover
 remote_desktop_password_insert_text_cb

This method is unused and was left behind after the rework in
commit 0ae59467fbd694d59fa367a0621de2482f169f44

Fixes #1703
---
 panels/sharing/cc-sharing-panel.c | 31 -------------------------------
 1 file changed, 31 deletions(-)

diff --git a/panels/sharing/cc-sharing-panel.c b/panels/sharing/cc-sharing-panel.c
index 65a7f616b..28e189bdc 100644
--- a/panels/sharing/cc-sharing-panel.c
+++ b/panels/sharing/cc-sharing-panel.c
@@ -1033,37 +1033,6 @@ cc_sharing_panel_check_schema_available (CcSharingPanel *self,
   return TRUE;
 }
 
-#define MAX_PASSWORD_SIZE 8
-static void
-remote_desktop_password_insert_text_cb (CcSharingPanel *self,
-                                        gchar          *new_text,
-                                        gint            new_text_length,
-                                        gpointer        position)
-{
-  int l, available_size;
-
-  l = gtk_entry_buffer_get_bytes (gtk_entry_get_buffer (GTK_ENTRY (self->remote_desktop_password_entry)));
-
-  if (l + new_text_length <= MAX_PASSWORD_SIZE)
-    return;
-
-  g_signal_stop_emission_by_name (self->remote_desktop_password_entry, "insert-text");
-  gtk_widget_error_bell (GTK_WIDGET (self->remote_desktop_password_entry));
-
-  available_size = g_utf8_strlen (new_text, MAX_PASSWORD_SIZE - l);
-  if (available_size == 0)
-    return;
-
-  g_signal_handlers_block_by_func (self->remote_desktop_password_entry,
-                                   (gpointer) remote_desktop_password_insert_text_cb,
-                                   self);
-  gtk_editable_insert_text (GTK_EDITABLE (self->remote_desktop_password_entry), new_text, available_size, position);
-  g_signal_handlers_unblock_by_func (self->remote_desktop_password_entry,
-                                     (gpointer) remote_desktop_password_insert_text_cb,
-                                     self);
-}
-#undef MAX_PASSWORD_SIZE
-
 static gboolean
 store_remote_desktop_credentials_timeout (gpointer user_data)
 {
-- 
2.34.1

